### PR TITLE
Attempt to reenable two FileStream tests for nano

### PR DESCRIPTION
**Pull request by danmosemsft**
It's not clear why they're disabled

### CODE REVIEW

**Comment by danmosemsft** [comment]
@dotnet-bot help please

**Comment by danmosemsft** [comment]
@mmitche how do I request Nano?

**Comment by mmitche** [comment]
@danmosemsft It always runs as part of the portable Windows tests.

**Comment by danmosemsft** [comment]
When marking Nano only (it wasn't you) we should comment why it is..

File: src/System.IO.FileSystem/tests/FileStream/ctor_str_fm_fa_fs_buffer_fo.cs [hunk 1]
```diff
@@ -64,7 +63,6 @@ public void ValidFileOptions(FileOptions option)
             }
         }
 
-        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
```
Comment (stephentoub) [inline src/System.IO.FileSystem/tests/FileStream/ctor_str_fm_fa_fs_buffer_fo.cs]: This didn't just disable for Nano... it disabled the tests entirely.  You need to replace the ConditionalTheory attributes with Theory.

Comment (danmosemsft) [inline src/System.IO.FileSystem/tests/FileStream/ctor_str_fm_fa_fs_buffer_fo.cs]: Ungghhhh

Comment (stephentoub) [inline src/System.IO.FileSystem/tests/FileStream/ctor_str_fm_fa_fs_buffer_fo.cs]: :smile: We've all been there.

**Pull request merged**